src/GoogleTrustedStores.php

```diff
@@ -21,14 +21,31 @@ public static function woocommerce_thankyou_order_received_text($text, $order) {
       return $text;
     }
     $delivery_date = date('Y-m-d', strtotime($order->get_date_created()->date('c') . ' + ' . $delivery_time . ' days'));
+
+    $product_gtins = [];
+    foreach ($order->get_items() as $item) {
+      $product_gtin = get_post_meta($item->get_product_id(), '_shop-standards_gtin', TRUE);
+      if (!empty($product_gtin)) {
+        $product_gtins[] = [
+          'gtin' => $product_gtin,
+        ];
```

Comment:
This will generate an associative array where multiple elements have the same key. Am I wrong? 🤔 

Something like:
```
$product_gtins[ 
  'gtin' => 'XXX',
  'gtin' => 'YYY',
];
```

Would not be better to just use an indexed array instead of an associative one?

Reply:
This was the task's requirement, it does seem a little strange to me, too.